.travis.yml

```diff
@@ -14,7 +14,7 @@ sudo: false  # use the new container-based Travis infrastructure
 dist: trusty # Ubuntu 14.04
 
 mono:
-  - latest
+  - nightly
```

Comment:
latest is currently completely broken, anyway... Running it would IMO just be a waste of cpu cycles